plugin.py

```diff
@@ -17,13 +17,13 @@ def plugin_info(self):
         plugin_name = ''.join(letter for letter in no_space_plugin_name if letter.isalnum() or letter == '_')
         plugin_description = f'Extracts "{file_format}" files and attaches its low-level data structure as a JSON text to the trace.'
         plugin_info = PluginInfo(
-            id=PluginId(domain='communityportal.hansken.org', category='extract', name=plugin_name),
+            id=PluginId(domain='{YOUR_ORGANISATION_DOMAIN}', category='extract', name=plugin_name),
             version='1.0.0',
             description=plugin_description,
-            author=Author('Jan', 'my@email.address', 'pim organisation'),
+            author=Author('{YOUR_NAME}', '{YOUR_EMAIL_ADDRESS}', '{YOUR_ORGANISATION}'),
             maturity=MaturityLevel.PROOF_OF_CONCEPT,
             webpage_url='',  # e.g. url to the code repository of your plugin
-            matcher='$data.mimeClass=archive',  # add the query for the firefli types of files your plugin should match
+            matcher='$data.fileType={FILETYPE_PROPERTY}',  # add the data stream fileType(s) of files your plugin should match
```

Comment:
Maybe also show the option of matching a file path? Maybe a comment that matching fileType is recommended (they should check the Supported Formats documentation to check if a fileType is recognized) but if not recognized that matching on file path can be used.

Reply:
The README also only mentiones that you can match on fileType. Maybe change is there (to)?

Reply:
Great suggestion. Improved the text a little and included a link to the documentation.